components/TextFields/src/MDCTextField.m

```diff
@@ -452,6 +452,17 @@ - (void)setText:(NSString *)text {
   }
 }
 
+- (void)setAttributedText:(NSAttributedString *)attributedText {
+  [super setAttributedText:attributedText];
+  [_fundament didSetText];
+
+  if (!self.isFirstResponder) {
```

Comment:
This is here because typing into the UITextField causes `setText:` to be called. I don't think that will happen with `attributedText` since the user can't input attributed strings.

Please test that and remove if you can, this check for `.isFirstResponder`.

Reply:
It looks like `-setAttributedText:` is called when the user finishes typing in a text field that previously held attributed text. In that case, it's only called when they did end editing (and `isFirstResponder` is true), but I'm hesitant to remove the check at this point.

Reply:
Deal.